### PR TITLE
Update layout.tsx

### PR DESCRIPTION
**Pull request by harshit040591**

### CODE REVIEW

**Comment by dwight-kaizenfox** [comment]
Pls fix test cases

File: src/app/layout.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { Inter } from 'next/font/google'
 const inter = Inter({ subsets: ['latin'] })
 
 export const metadata: Metadata = {
-  title: 'Create Next App',
+  title: 'Create Next App!',
```
Comment (dwight-kaizenfox) [inline src/app/layout.tsx:8]: This is probably what broke the test case

**Review by john-kaizenfox — APPROVED** [review]

**Review by john-kaizenfox — APPROVED** [review]

**Review by dwight-kaizenfox — CHANGES_REQUESTED** [review]

**Review by john-kaizenfox — CHANGES_REQUESTED** [review]
Fix the breaks

**Comment by harshit040591** [comment]
> 

Completed

**Pull request closed without merging**